packages/product/src/services/product.ts

```diff
@@ -1,11 +1,23 @@
-import { Context, DAL, FindConfig, ProductTypes } from "@medusajs/types"
+import {
```

Comment:
Hard to remove this file, because of all the rearrangement. I think we should get this to land on `develop` now, and then we can always clean up (if needed), as we use the module.

Reply:
Yeah it was a patch I decided to pull now, there was no better way around it. I'll get this merged ASAP 👍